frontend/packages/rhoas-plugin/src/components/service-list/ServiceListPage.tsx

```diff
@@ -0,0 +1,137 @@
+import * as React from 'react';
+import NamespacedPage, {
+  NamespacedPageVariants,
+} from '@console/dev-console/src/components/NamespacedPage';
+import { history, LoadingBox } from '@console/internal/components/utils';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { useActiveNamespace } from '@console/shared';
+import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
+import ServiceInstance from './ServiceInstance';
+import { CloudServicesRequestModel } from '../../models/rhoas';
+import { ServicesRequestCRName } from '../../const';
+import {
+  createKafkaConnection,
+  createCloudServicesRequestIfNeeded,
+  deleteKafkaConnection,
+  listOfCurrentKafkaConnectionsById,
+} from '../../utils/resourceCreators';
+import { KafkaRequest } from '../../utils/rhoas-types';
+import {
+  isResourceStatusSuccessfull,
+  isAcccesTokenSecretValid,
+  getFinishedCondition,
+} from '../../utils/conditionHandler';
+import { ServicesErrorState } from '../states/ServicesErrorState';
+import { useTranslation } from 'react-i18next';
+
+const ServiceListPage = () => {
+  const [currentNamespace] = useActiveNamespace();
+  const [selectedKafka, setSelectedKafka] = React.useState<number>();
+  const [currentKafkaConnections, setCurrentKafkaConnections] = React.useState<string[]>([]);
+  const { t } = useTranslation();
+  const [kafkaCreateError, setKafkaCreateError] = React.useState<string>();
+
+  React.useEffect(() => {
+    const createKafkaRequestFlow = async () => {
+      await createCloudServicesRequestIfNeeded(currentNamespace);
+
+      const currentKafka = await listOfCurrentKafkaConnectionsById(currentNamespace);
+      if (currentKafka) {
+        setCurrentKafkaConnections(currentKafka);
+      }
+    };
+    createKafkaRequestFlow();
+  }, [currentNamespace]);
```

Comment:
Operator doesn't provide refresh capabilities because of the API limitations. Once we get results they will stay the same until we patch existing request.